src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1050,6 +1050,25 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "Home_Alert_RiskStatusLowered_PrimaryButtonTitle" = "OK";
 
+/* Contact Diary Information */
```

Comment:
There is a section at the bottom of the File for all Strings related to ContactDiary, would be nice to cluster them :)

Reply:
I moved them